libraries/freertos_plus/standard/freertos_plus_tcp_multi/include/FreeRTOS_DNS.h

```diff
@@ -157,14 +157,21 @@ uint32_t ulDNSHandlePacket( NetworkBufferDescriptor_t *pxNetworkBuffer );
 
 /*
  * Lookup a IPv4 node in a blocking-way.
+ * It returns a 32-bit IP-address, 0 when not found.
+ * gethostbyname() is already deprecated.
  */
 uint32_t FreeRTOS_gethostbyname( const char *pcHostName );
 
+/* getaddrinfo() replaces gethostbyname(). It can also return IPv6 addresses when enabled. */
```

Comment:
Wrote a more complete comment about the new function `FreeRTOS_getaddrinfo()`

~~~c
/*
 * FreeRTOS_getaddrinfo() replaces FreeRTOS_gethostbyname().
 * When 'ipconfigUSE_IPv6' is defined, it can also retrieve IPv6 addresses,
 * in case pxHints->ai_family equals FREERTOS_AF_INET6.
 * Otherwise, or when pxHints is NULL, only IPv4 addresses will be returned.
 */
~~~